data/plugins.json

```diff
@@ -20,11 +20,11 @@
     "node": 6
   },
   "transform-es2015-classes": {
-    "chrome": 46,
+    "chrome": 45,
     "edge": 13,
     "firefox": 45,
     "safari": 10,
-    "node": 5
+    "node": 4
```

Comment:
Node 4 doesn't support spread, so non-transformed classes that have `super(...args)` will not run.

```js
class X extends Y {
  constructor (...args) {
    super(...args)
  }
}
```

is invalid in node 4 regardless of the other transforms. That example constructor is useless as that is the default implementation, but when `transform-class-properties` runs, it has to generate a constructor with that content to attach the public property initializers:

```js
class X extends Y {
  z = null
}
```

transforms to:

```js
class X extends Y {
  constructor (...args) {
    super(...args)
    this.z = null
  }
}
```